source/features/view-markdown-source.tsx

```diff
@@ -10,9 +10,7 @@ import fetchDom from '../libs/fetch-dom';
 import * as icons from '../libs/icons';
 import {blurAccessibly} from './comment-fields-keyboard-shortcuts';
 
-const linkedSourceDom = Symbol('Attached RGH source dom');
-const linkedRenderedDom = Symbol('Attached RGH rendered dom');
-const rghButtons = new WeakMap<Symbol, Element>();
+const rghButtons = new WeakMap<Element, Element>();
```

Comment:
I believe I implemented this functionality properly with a WeakMap. Mind just giving it a double check?